.github/actions/lint_php-lint.sh

```diff
@@ -22,5 +22,4 @@ echo "Run code static analysis"
 vendor/bin/phpstan analyze \
   --ansi \
   --memory-limit=2500M \
-  --no-interaction \
-  --no-progress
+  --no-interaction
```

Comment:
Why removing the `--no-progress`? Display on GH actions does not convince me.

Reply:
Why not using the local configuration file then? That's exactly the point of this pull request, no?

Reply:
(on my local instance, I do not want to see progress most of the time)

Reply:
I believe "--no-progresss" is a command line option, not a configuration one (and thus can't be set from the config file).

I am unsure why you do not want to see progress on your local instance, it is just one additional line being displayed (besides, we already display progress for almost all others linters).

IMO being able to see the progression of a long process would be upside, especially when it runs for 15 minutes using the default configuration.

Reply:
>  (besides, we already display progress for almost all others linters).

We usually use output when the output is correct... 

Anyway. It seem you  do really want to add this output.. So , do what you want; I've already spend too much time on such an useless thing.

Reply:
> We usually use output when the output is correct...

Maybe there is a misunderstanding, in the context we are discussing (local machine) the output is correct.

Its only the CI report interface that has the weird display because it isn't able to deal with the fact that the progress bar is redrawing itself.